packages/overlay/src/utils/instrumentation.ts

```diff
@@ -0,0 +1,62 @@
+import * as Sentry from '@sentry/react';
+import { createRoutesFromChildren, matchRoutes, useLocation, useNavigationType } from 'react-router-dom';
+import { useEffect } from '../react-instance';
+
+export default function initSentry(initialTab: string, options: Sentry.BrowserOptions = {}) {
+  const sentryTraceParent: Record<string, string> = {};
+  const navTiming = performance.getEntriesByType('navigation');
+  if (navTiming.length > 0) {
+    const serverTiming = (navTiming[0] as PerformanceNavigationTiming).serverTiming;
+    if (serverTiming && serverTiming.length > 0) {
+      for (const { name, description } of serverTiming) {
+        if (name === 'sentryTrace' || name === 'baggage') {
+          sentryTraceParent[name] = description;
+        }
+      }
+    }
+  }
+  const hasTraceParent = Object.keys(sentryTraceParent).length === 2;
+  const integrations = [
+    // See docs for support of different versions of variation of react router
+    // https://docs.sentry.io/platforms/javascript/guides/react/configuration/integrations/react-router/
+    Sentry.reactRouterV6BrowserTracingIntegration({
+      instrumentPageLoad: !hasTraceParent,
```

Comment:
can we always default to `false` here? Further below we're specifically calling `startBrowserTracingPageLoadSpan` anyway? Again, I'm very concerned about interference so this would only be relevant if we don't interfere at all.

Reply:
I'm also guessing the concern is when we are in overlay mode. In that case we don't even call this function so we should be safe.

The reason this is conditional is due to some browser compat edge case where my `server-timings` hack may not work so we may not be able to get a trace parent.

Reply:
yup, that was the concern. I think this can be disregarded now that I know we only enable sentry in standalone mode